modules/javafx.controls/src/test/java/test/javafx/scene/control/TreeTableViewTest.java

```diff
@@ -786,6 +786,18 @@ private void verifySelectionAfterPermutation() {
         });
     }
 
+    @Test public void testSetSortOrderretainsWhenRootHasNoChildren() {
```

Comment:
Minor, fix the camel case: `testSetSortOrderRetainsWhenRootHasNoChildren`

Reply:
Thanks, it is corrected now.